ebean-core/src/main/java/io/ebeaninternal/server/querydefn/HashCodeBindHash.java

```diff
@@ -0,0 +1,63 @@
+package io.ebeaninternal.server.querydefn;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import io.ebeaninternal.api.BindHash;
+
+/**
+ * HashCode builder that uses Object.hashCode for computing bind-hashes.
+ * This is a fast and lightweight implementation, but may produce collisions.
+ *
+ * @author Roland Praml, FOCONIS AG
+ */
+public class HashCodeBindHash implements BindHash {
+
+  private final List<Object> values = new ArrayList<>();
```

Comment:
Good idea, this would make MD5 complete obsolete.
BTW: we do not need a separate "hashCode" computation, we can use just `values.hashCode()`